app/util/passcode-creator.ts

```diff
@@ -28,11 +28,9 @@ class PasscodeCreator {
   }
 
   createPasscode(): string {
-    return [
-      this.selectRandomWord(),
-      this.selectRandomWord(),
-      this.selectRandomWord()
-    ].join(this.separator);
+    return [...Array(PASSCODE_LENGTH).keys()]
```

Comment:
Small detail, and I might be wrong, but I don't think spreading the keys into a new Array has any effect here